File: Component/DSDynamicView.m
//
//  DSDynamicView.m
//  DynamicScrollView
//
//  Created by Damir Stuhec on 29/06/14.
//  Copyright (c) 2014 damirstuhec. All rights reserved.
//

#import "DSDynamicView.h"

@implementation DSDynamicView

- (id)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
    }
    return self;
}

- (BOOL)isEqual:(id)object
{
    // override in subclass
    return [super isEqual:object];
}

- (NSComparisonResult)compareToObject:(id)object
{
    // override in subclass
    return NSOrderedDescending;
}

@end

File: Component_Swift/DSDynamicView.swift
//
//  DSDynamicView.swift
//  DynamicScrollView
//
//  Created by Damir Stuhec on 03/10/14.
//  Copyright (c) 2014 damirstuhec. All rights reserved.
//

import UIKit

class DSDynamicView : UIScrollView
{
    override func isEqual(object: AnyObject?) -> Bool
    {
        // override in subclass
        return super.isEqual(object)
    }
    
    func compareToObject(object: AnyObject?) -> Bool
    {
        // override in subclass
        return false
    }
}